### PR TITLE
Check pTexName for nullptr, fix wrongly height->width

### PR DESCRIPTION
**Pull request by Jupeyy**
thanks to @ChillerDragon for finding this

## Checklist

- [x] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test if it works standalone, system.c especially
- [x] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [x] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: src/engine/client/graphics_threaded.cpp [hunk 1]
```diff
@@ -496,7 +496,7 @@ IGraphics::CTextureHandle CGraphics_Threaded::LoadTextureRaw(int Width, int Heig
 	void *pTmpData = malloc(MemSize);
 	if(!ConvertToRGBA((uint8_t *)pTmpData, (const uint8_t *)pData, Width, Height, Format))
 	{
-		dbg_msg("graphics", "converted image %s to RGBA, consider making its file format RGBA", pTexName);
+		dbg_msg("graphics", "converted image %s to RGBA, consider making its file format RGBA", pTexName ? pTexName : "(no name)");
```
Comment (Chairn) [inline src/engine/client/graphics_threaded.cpp:499]: To be honest, "(null)" already says it's an unnamed image

Comment (Jupeyy) [reply src/engine/client/graphics_threaded.cpp:499]: this isnt standard, nullptr is UB

Comment (Chairn) [reply src/engine/client/graphics_threaded.cpp:499]: ah true, actually it should have segfaulted

**Pull request merged**